script.js

```diff
@@ -220,21 +223,33 @@ window.addEventListener('load', () => {
       e.preventDefault();
       return;
     }
-    const name = document.getElementById('name');
+
     if (!name.validity.valid) {
       // submit form
       document.getElementById('error').innerHTML = 'Invalid Name';
       e.preventDefault();
       return;
     }
-    const text = document.getElementById('text');
+
     if (!text.validity.valid) {
       // submit form
       document.getElementById('error').innerHTML = 'Invalid Text';
       e.preventDefault();
       return;
     }
-    // show errors
+
+    const form = document.getElementById('contact-form');
+    const myFormData = new FormData(form);
+    const formDataObj = Object.fromEntries(myFormData.entries());
+    localStorage.setItem('contactform', JSON.stringify(formDataObj));
```

Comment:
- [ ] When the user enter values into the input fields, the form data for all inputs should be saved to a single local storage registry, which currently is not the case. Kindly fix this

Reply:
Thank You @Munsa1. We will correct that